sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
Is this being introduced for the first time because of `WritableCoder` moving into `hadoop-common` ? I guess because of `Coder`, right ?